TPM-Commands/TPM2_StartAuthSession.md

```diff
@@ -54,26 +70,30 @@ The `sessionType` input parameter must be one of:
  - `TPM_SE_POLICY`
  - `TPM_SE_TRIAL`
 
-### HMAC Sessions
+## HMAC Sessions
```

Comment:
Should these stay at level three indentation under `Session Types`?

Reply:
I up-leveled all those sub-sections.  Somehow it seemed cleaner to me that way.